documentation/cloud-manager-guide/admin/account-settings/update-account.md

```diff
@@ -0,0 +1,79 @@
+---
+# Account Settings
+
+This guide provides an overview of how to update your account settings and manage your API tokens.
+(**confirming with RL where these details appear/what they are used for, ie. just for display on *Users* tab? For EOC communications? Communications with RL tech support?**
+
+**Also confirming API details, what they are used for, etc.** ).
+
+## Getting started
+
+From any tab in the *Admin* view, your account avatar (![image description](../images/icon-avatar.png)) will be visible in the upper right corner of the screen. To access your account settings, select the avatar icon (![image description](../images/icon-avatar.png)) which will expand the account dropdown menu. From the dropdown menu, select **Account Settings** to open the *Account Settings* screen.
+
+![image description](images/account-select-settings.png)
+
+## Account settings
+
+From the *Account Settings* screen you can update your user details including your first name, last name, email address associated with the account, and your profile image. 
+
+### Update user details
+
+To update your first name, last name, or email address, enter your information in the spaces provided and select **Save** to update.
+
+![image description](images/account-userdetails.png)
+
+### Update profile image
+
+To update your profile image, select "Edit Avatar"... (**confirming with RL if this opens a drag and drop screen or if it just opens the user's local file system to upload, and if there are any image size limits**).
+
+![image description](images/account-edit-avatar.png)
+
+## Manage API tokens
+
+### Create an API token
+
+To create a new API token, select the **Generate** (![image description](images/icon-generate.png)) button located just above the *API token* input field. This will generate a unique API token that populates in the *API token* input field.
+
+![image description](images/account-select-generate.png)
+
+Set the API token expiration date by selecting a duration from the *Expiration Date* dropdown menu and select **Save** to create the new API token.
+
+![image description](images/account-expiration-dropdown.png)
+
+You will receive an expiration warning when your API token is close to expiring. A warning is displayed in your *Account Settings* next to the *Expiration Date* dropdown and ... (**confirming with RL when user receives first notification and where warnings appear**). You will receive an expiration reminder every day starting *X amount of time* before your API token is set to expire.
+
+> **Warning:** API token expiration is a permanent action. Once a token has expired it cannot be retrieved.
+
+![image description](images/account-expiration-warning.png)
+
+### Share an API token
+
+To copy the API token to share, select the **Copy to Clipboard* icon (![image description](images/icon-copy.png)) located next to the *API Token* input field and proceed to share through your preferred channel.
```

Comment:
"To copy the API token, select the *Copy to Clipboard* icon."

There's not really a need to "share" it per se - they're likely just copying it to use within their calls.